addons/binding/org.openhab.binding.velux/ESH-INF/i18n/velux_de.properties

```diff
@@ -0,0 +1,113 @@
+#/**
```

Comment:
This file normally does not have a copyright and this copyright is non default.

Reply:
Adapted